src/Magnum/Platform/GlfwApplication.h

```diff
@@ -436,6 +437,33 @@ class GlfwApplication {
         /** @copydoc Sdl2Application::redraw() */
         void redraw() { _flags |= Flag::Redraw; }
 
+    public:
+        enum class Cursor : UnsignedInt {
+            Arrow,      /**< Arrow */
+            IBeam,      /**< I-Beam */
+            Crosshair,  /**< Crosshair */
+            ResizeWE,   /**< Double arrow pointing west and east */
+            ResizeNS,   /**< Double arrow pointing north and south */
+            Hand        /**< Hand */
+        };
```

Comment:
Instead of having the switch, I think the values could go directly in here:

```cpp
Arrow = GLFW_ARROW_CURSOR,
IBeam = GLFW_IBEAM_CURSOR,
...
```

Same for SDL. All other Application enums (buttons, keys, ...) are done this way to avoid the extra switch overhead, so this would be then consistent :)

Reply:
Well, initially I made this and it worked for SDL but not for glfw, the reason is that glfw stores these flags as #define 0x0003600x, so casting e.g. a Cursor::Arrow to UnsignedInt it didn't gave me a 0-based index like Sdl does so it was accessing unallocated memory.
But I think I can fix this doing `UnsignedInt index = 0x00036000 - std::static_cast<UnsignedInt>(Cursor::Arrow)` to get the index to save the value to the array

Reply:
Ahhhhhh you're right, then the easy indexing into the array wouldn't work. Keep it like you have it now, then, and instead of a switch you could do

```cpp
constexpr Int CursorMap[] {
    GLFW_ARROW_CURSOR,
    GLFW_IBEAM_CURSOR,
    ...
};
```

and then do the conversion as `CursorMap[UnsignedInt(cursor)]`. I'm [doing a similar thing](https://blog.magnum.graphics/backstage/gfx-api-enum-mapping/) for the PixelFormat enum and conversion to GL/Vulkan.

Reply:
Dang, too late to reply :sweat_smile:

Reply:
> Dang, too late to reply 😅

Literally when I clicked Enter to push the changes I heard the *bling* that notified me of your comment :D

Btw I'll change it again, no problem.